ols/app/endpoints/health.py

```diff
@@ -7,28 +7,27 @@
 
 from fastapi import APIRouter
 
-from ols.app.models.models import HealthResponse
+from ols import config
+from ols.app.models.models import HealthResponse, ReadinessResponse
 
 router = APIRouter(tags=["health"])
 
 
-# TODO: OLS-488 Define behaviour of /readiness and /liveness endpoints in OLS service
-#
-# Example status response:
-# {
-#     "status": "unhealthy",
-#     "version": "1.0.0",
-#     "dependencies": {
-#         "database": "healthy",
-#         "externalApi": "unhealthy"
-#     }
-# }
+def index_is_ready() -> bool:
+    """Check if the index is loaded."""
+    if config._rag_index is None:
```

Comment:
@onmete When there is no RAG content  configured , the server can proceed without RAG content . In that case the readiness can return false .

Reply:
Right, updated.